example/newfilewithappauth/main.go

```diff
@@ -0,0 +1,96 @@
+//Copyright 2021 The go-github AUTHORS. All rights reserved.
+
+//Use of this source code is governed by a BSD-style
+//license that can be found in the LICENSE file.
+
+//newfilewithappauth demonstrates the functionality of Github's app authentication
+//methods by fetching an installation access token and reauthenticating to Github
+//with OAuth configurations.
+
+package main
```

Comment:
Here's what these lines should look like to be consistent with the rest of the repo:

```go
// Copyright 2021 The go-github AUTHORS. All rights reserved.
//
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// newfilewithappauth demonstrates the functionality of GitHub's app authentication
// methods by fetching an installation access token and reauthenticating to GitHub
// with OAuth configurations.
package main
```